ruby-training/trainees/alan/test_katas/01/math_maddness/math_maddness.rb

```diff
@@ -0,0 +1,26 @@
+require 'rspec/autorun'
+
+class Aritmetic_operations
```

Comment:
Class names should be CamelCase... In this case, `ArithmeticOperations`. Also, please watch the spelling (missing the "h" in arithmetic).

Reply:
Also, the file name should match the class name in snake_case (`arithmetic_operations.rb`) when writing a single class per file.